### PR TITLE
Authentication system

### PR DESCRIPTION
**Pull request by upayanmazumder**
Features an authentication system in firebase

### CODE REVIEW

File: qwik/src/routes/auth/signup/index.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+interface SignupResponse {
+  error?: string;
+  [key: string]: any;
+}
+
+export const usePostSignupAction = routeAction$(async (props): Promise<SignupResponse> => {
+  console.log('usePostSignupAction', props);
+  const { email, password } = props;
+  console.log('email', email);
+  console.log('password', password);
```
[the remark this answers is not in the document]
Comment (upayanmazumder) [reply qwik/src/routes/auth/signup/index.tsx:13]: high priority

[the remark this answers is not in the document]
Comment (dave9123) [reply qwik/src/routes/auth/signup/index.tsx:13]: That was for debugging lol

[the remark this answers is not in the document]
Comment (dave9123) [reply qwik/src/routes/auth/signup/index.tsx:13]: I'll delete the console.log part

**Comment by upayanmazumder** [comment]
alr we will merge this when we have a proper signup & login system implemented 👍

**Comment by dave9123** [comment]
Doesn't seem to fit well, as Qwik and API is running on a different place

**Comment by dave9123** [comment]
What I'm thinking, is just go over with an actual database

**Comment by upayanmazumder** [comment]
> What I'm thinking, is just go over with an actual database

wdym?

**Comment by Larssies** [comment]
> > What I'm thinking, is just go over with an actual database
> 
> wdym?

We are working on this, itll be fixed.

**Pull request merged**